content/sensu-go/5.15/api/apikeys.md

```diff
@@ -0,0 +1,152 @@
+---
+title: "APIKeys API"
+description: "The APIKeys API provides HTTP access to API key data. This is a reference for the APIKeys API in Sensu Go, including examples for returning lists of API keys, creating API keys, and more. Read on for the full reference."
+version: "5.15"
+product: "Sensu Go"
+menu:
+  sensu-go-5.15:
+    parent: api
+---
+
+- [The `/apikeys` API endpoint](#the-apikeys-api-endpoint)
+	- [`/apikeys` (GET)](#apikeys-get)
+	- [`/apikeys` (POST)](#apikeys-post)
+- [The `/apikeys/:apikey` API endpoint](#the-apikeysapikey-api-endpoint)
+	- [`/apikeys/:apikey` (GET)](#apikeysapikey-get)
+	- [`/apikeys/:apikey` (DELETE)](#apikeysapikey-delete)
+
+
+## The `/apikeys` API endpoint
+
+### `/apikeys` (GET)
+
+The `/apikeys` GET endpoint retrieves all API keys.
+
+#### EXAMPLE {#apikeys-get-example}
+
+The following example demonstrates a request to the `/apikeys` API, resulting in a JSON array containing all API keys.
+
+{{< highlight shell >}}
+curl http://127.0.0.1:8080/api/core/v2/apikeys -H "Authorization: Bearer $SENSU_TOKEN"
+{
+  "username": "admin"
+}
+
+HTTP/1.1 200 OK
+[
+  {
+    "metadata": {
+      "name": "83abef1e-e7d7-4beb-91fc-79ad90084d5b"
+    },
+    "username": "admin",
+    "created_at": 1570640363
+  }
+]
+{{< /highlight >}}
+
+#### API Specification {#apikeys-get-specification}
+
+/apikeys (GET)  | 
+---------------|------
+description    | Returns the list of API keys.
+example url    | http://hostname:8080/api/core/v2/apikeys
+pagination     | This endpoint supports pagination using the `limit` and `continue` query parameters. See the [API overview][1] for details.
+response type  | Array
+response codes | <ul><li>**Success**: 200 (OK)</li><li>**Error**: 500 (Internal Server Error)</li></ul>
+output         | {{< highlight shell >}}
+[
+  {
+    "metadata": {
+      "name": "83abef1e-e7d7-4beb-91fc-79ad90084d5b"
+    },
+    "username": "admin",
+    "created_at": 1570640363
+  }
+]
+{{< /highlight >}}
+
+### `/apikeys` (POST)
+
+/apikeys (POST) | 
```

Comment:
We should add a field to this table about response headers. For this endpoint, it returns a response header `Location` which contains the relative path to the new api key, ex: `/api/core/v2/apikeys/f28caef8-a3f2-4c7d-b722-affce2815af9`.

Reply:
Added location in output (not sure if correct) and clarified in description